packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -459,15 +472,19 @@ class _FocusState extends State<Focus> {
   @override
   Widget build(BuildContext context) {
     _focusAttachment.reparent();
+    Widget semantics;
+    if (widget.includeSemantics) {
+      semantics = Semantics(
+        focusable: _canRequestFocus,
+        focused: _hasPrimaryFocus,
+        child: widget.child,
+      );
+    } else {
+      semantics = widget.child;
+    }
```

Comment:
In many other places of the framework we use this pattern for these kinda things:

```dart
Widget child = widget.child;
if (widget.includeSemantics) {
  child = Semantics(
     // ...
    child: child,
   );
}
```

Reply:
Thanks.  Updated to match that pattern.